app/controllers/dams_resource_controller.rb

```diff
@@ -30,6 +30,17 @@ def show
     # get metadata from solr
     @document = get_single_doc_via_search(1, {:q => "id:#{params[:id]}"} )
 
+    # generate facet collection list for collection page only
+    if !@document.nil?
```

Comment:
Done. I see it break the test in https://github.com/ucsdlib/damspas/blob/develop/spec/features/dams_object_spec.rb#L371-L374 with the 404 status so just keeping the conditional check unless we really want the 404 status. See commit https://github.com/ucsdlib/damspas/commit/afdfccad8fec5c37a1e1d88b99699bb108316518